solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -561,6 +561,16 @@ contract RandomBeacon is IRandomBeacon, IApplication, Ownable {
         );
     }
 
+    /// @notice Withdraw rewards for the given staking provider to their
+    ///         beneficiary address. Reverts if staking provider has not
+    ///         registered the operator address.
+    function withdrawRewards(address stakingProvider) external {
+        address operator = stakingProviderToOperator(stakingProvider);
+        require(operator != address(0), "Unknown operator");
+        (, address beneficiary, ) = staking.rolesOf(stakingProvider);
+        sortitionPool.withdrawRewards(operator, beneficiary);
```

Comment:
Would it make sense to add a requirement that `beneficiary != address(0)`? Maybe it should be a part of `sortitionPool.withdrawRewards`?